### PR TITLE
{tools}[foss/2018b] awscli v1.16.290 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by timeu**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/a/awscli/awscli-1.16.290-foss-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+name = 'awscli'
+version = '1.16.290'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/awscli'
+description = 'Universal Command Line Environment for AWS'
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
```
Comment (verdurin) [inline easybuild/easyconfigs/a/awscli/awscli-1.16.290-foss-2018b-Python-3.6.6.eb]: @timeu Any reason not to use `pip` here?

Comment (timeu) [reply easybuild/easyconfigs/a/awscli/awscli-1.16.290-foss-2018b-Python-3.6.6.eb]: No, will fix it. I copy and pasted the old verison and forgot it

**Comment by boegel** [comment]
This is the list of downloaded dependencies that are detected when `use_pip` is disabled but `download_dep_fail` is enabled:

```
Downloading https://files.pythonhosted.org/packages/4f/a6/728666f39bfff1719fc94c481890b2106837da9318031f71a8424b662e12/colorama-0.4.1-py2.py3-none-any.whl#sha256=f8ac84de7840f5b9c4e3347b3c1eaa50f7e49c2b07596221daec5edaabbd7c48
Downloading https://files.pythonhosted.org/packages/e3/e8/b3212641ee2718d556df0f23f78de8303f068fe29cdaa7a91018849582fe/PyYAML-5.1.2.tar.gz#sha256=01adf0b6c6f61bd11af6e10ca52b7d4057dd0be0343eb9283c878cf3af56aee4
Downloading https://files.pythonhosted.org/packages/16/8a/1fc3dba0c4923c2a76e1ff0d52b305c44606da63f718d14d3231e21c51b0/s3transfer-0.2.1-py2.py3-none-any.whl#sha256=b780f2411b824cb541dbcd2c713d0cb61c7d1bcadae204cdddda2b35cef493ba
Downloading https://files.pythonhosted.org/packages/e1/ae/baedc9cb175552e95f3395c43055a6a5e125ae4d48a1d7a924baca83e92e/rsa-3.4.2-py2.py3-none-any.whl#sha256=43f682fea81c452c98d09fc316aae12de6d30c4b5c84226642cf8f8fd1c93abd
Downloading https://files.pythonhosted.org/packages/22/cd/a6aa959dca619918ccb55023b4cb151949c64d4d5d55b3f4ffd7eee0c6e8/docutils-0.15.2-py3-none-any.whl#sha256=6c4f696463b79f1fb8ba0c594b63840ebd41f059e92b31957c46b74a4599b6d0
Downloading https://files.pythonhosted.org/packages/8a/93/ea2ec042794dfda186348df02c6057223a8bbc21c055124fbe3e16925441/botocore-1.13.26-py2.py3-none-any.whl#sha256=9fefb42c6d4fa0079a52b49e5491fa0738cca63649f68be180b3ed6c253d2622
Downloading https://files.pythonhosted.org/packages/83/94/7179c3832a6d45b266ddb2aac329e101367fbdb11f425f13771d27f225bb/jmespath-0.9.4-py2.py3-none-any.whl#sha256=3720a4b1bd659dd2eecad0666459b9788813e032b83e7ba58578e48254e0a0e6
```

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/4e4ec213d32993981f28461912d01715 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @timeu!

File: easybuild/easyconfigs/a/awscli/awscli-1.16.290-foss-2018b-Python-3.6.6.eb [hunk 2]
```diff
@@ -9,16 +9,43 @@ description = 'Universal Command Line Environment for AWS'
...
 use_pip = True
-download_dep_fail = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('colorama', '0.4.1', {
+        'checksums': ['05eed71e2e327246ad6b38c540c4a3117230b19679b875190486ddd2d721422d'],
+    }),
+    ('PyYAML', '5.1.2', {
```
Comment (boegel) [inline easybuild/easyconfigs/a/awscli/awscli-1.16.290-foss-2018b-Python-3.6.6.eb]: @timeu Please use a `PyYAML` dependency instead (there's an easyconfig for `PyYAML` 3.13 with this toolchain)

The `PyYAML` we provide depends on the `libyaml` C library, which results in a lot faster YAML parsing than a Python-only installation...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2404.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/8f265a3a8654689c5227553d30fb06a2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/324f134d7dc45c71bef0b22b25f760ec for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3169.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/52154c66ed67e55e74af1984ede549c9 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @timeu!

**Pull request merged**